File: python/LinkedList/collection/utils/uimanager.py
from ..ds.linkedlist import LinkedList
from .filemanager import FileManager 

class UIManager:

    def showMenu(self):
        print("1. Insert")
        print("2. Search")
        print("3. Delete")
        print("4. Display")
        print("5. Save to file")
        print("6. Exit")

    def acceptChoice(self):
        return int(input("Enter your choice: "))
        

File: C/LinkedList/src/uimanager.c
#include "../include/uimanager.h"

void showMenu(){
    printf("\n1. Insert Item\n");
    printf("2. Search Item\n");
    printf("3. Delete Item\n");
    printf("4. Display List\n");
    printf("5. Save and Exit\n");
}

int acceptChoice(){
    int choice;
    printf("Enter your choice: ");
    scanf("%d", &choice);
    return choice;
}
    